src/components/Header/Header.module.scss

```diff
@@ -1,6 +1,145 @@
 @import "../../styles/var.scss";
 
 .header {
-	font-size: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px;
+  background-color: #f8f9fa;
+  font-size: 16px;
+}
+
+.container {
+  width: 100%;
+  max-width: 1200px;
```

Comment:
absolute values for these must be inside media queries
otherwise, please use relative units, e.g.: rem, em, %